src/app/ui/components/Account/AccountDetails/BonusPlaces/BonusPlacesBadge.py

```
@@ -30,10 +30,11 @@ class BonusPlacesBadge(QPushButton):
 
     placeEdited: pyqtSignal = pyqtSignal(int)
 
-    def __init__(self, symbol, completion_status, index, bonus_place_id=None, name=None, current=False, function=None, *args, **kwargs):
+    def __init__(self, symbol, completion_status, index, bonus_place_id=None, name=None, current=False, function=None, account_id=None, *args, **kwargs):
```

Pylint (3.3.7)

[refactor] Too many arguments (9/5)

(R0913)